### PR TITLE
Primes cm

### PR DESCRIPTION
**Pull request by morecallan**
@jcockhren @katerebekah @BlaiseGratton @guthb

### CODE REVIEW

File: Mathmagician/Primes.cs [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Mathmagician
+{
+    public class Primes
+    {
+
+        public static bool isPrime(int number)
```
Comment (katerebekah) [inline Mathmagician/Primes.cs:12]: You should unit test this method, too

File: Mathmagician/Primes.cs [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        }
+
+        public List<int> ReturnPrimeNumbersList(int userInput)
+        {
+            List<int> primesList = new List<int>();
+            int i = 0;
+            int j = 0;
+
+            while (i < userInput)
+            {
```
Comment (katerebekah) [inline Mathmagician/Primes.cs:35]: This is very good, clean code for checking primes. It's well thought out and logically separated.  Good job!

**Comment by katerebekah** [comment]
Once you add a unit test for the isPrime method, I'll thumbs this up

**Comment by katerebekah** [comment]
👍

**Comment by jcockhren** [comment]
Did you review this @guthb? I see no evidence so such. @morecallan, why did you merge this if your partner hasn't given you the ok?

Now that this has been merged without your team :eyes: at it, you have merge conflicts on the #9 Pull Request.

**Comment by guthb** [comment]
I did not,, and think this was just a simple goof as we had some system issues last night with Callan's vm and the project files, we just got in a hurry.  I do have conflicts now but i am looking forward to working on my git-fu on how to correct those issues with VS2015.

**Pull request merged**